From 692969eb3667aa0240828b6fb08f5aa78c8a3c4c Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Thu, 2 Apr 2020 14:47:55 +0800
Subject: [PATCH] ARM: dts: rockchip: Change tsadc clk 2M to 4M for rv1126

TSADC PHY clk is 2~6M, but a few boards 2M tsadc does not work.

Change-Id: I34b7e845be3713d9603ec159eae5f57cf63b003a
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm/boot/dts/rv1126.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index afc3de924b47..89b7ca86b51b 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -1104,7 +1104,7 @@
 		rockchip,grf = <&grf>;
 		interrupts = <GIC_SPI 39 IRQ_TYPE_LEVEL_HIGH>;
 		assigned-clocks = <&cru CLK_CPU_TSADC>;
-		assigned-clock-rates = <2000000>;
+		assigned-clock-rates = <4000000>;
 		clocks = <&cru CLK_CPU_TSADC>, <&cru PCLK_CPU_TSADC>,
 			 <&cru CLK_CPU_TSADCPHY>;
 		clock-names = "tsadc", "apb_pclk", "phy_clk";
@@ -1122,7 +1122,7 @@
 		rockchip,grf = <&grf>;
 		interrupts = <GIC_SPI 113 IRQ_TYPE_LEVEL_HIGH>;
 		assigned-clocks = <&cru CLK_NPU_TSADC>;
-		assigned-clock-rates = <2000000>;
+		assigned-clock-rates = <4000000>;
 		clocks = <&cru CLK_NPU_TSADC>, <&cru PCLK_NPU_TSADC>,
 			 <&cru CLK_NPU_TSADCPHY>;
 		clock-names = "tsadc", "apb_pclk", "phy_clk";
-- 
2.35.3

